.circleci/continue-config.yml

```
@@ -84,6 +83,16 @@ jobs:
       - store_test_results:
           path: './reports'
 
+  publish-to-chromatic: # Publish the baseline chromatic build from master
+    executor: vfcommon/node-executor-node-20
+    steps:
+      - attach_workspace:
+          at: ~/voiceflow
+      - run:
+          name: Publish to Chromatic
+          working_directory: packages/react-chat
+          command: yarn chromatic:publish --exit-zero-on-changes --exit-once-uploaded
```

I think the flags here should actually be `--auto-accept-changes --exit-once-uploaded`
```suggestion
          command: yarn chromatic:publish --auto-accept-changes --exit-once-uploaded
```

---

oh no, trusted copilot too much